packages/api/lib/token-set.ts

```diff
@@ -0,0 +1,20 @@
+export class TokenSet {
+  public expiresAt: number;
+  public access_token: string;
+
+  constructor(values) {
+    Object.assign(this, values);
+  }
+
+  set expires_in(value) {
+    this.expiresAt = Date.now() + Number(value);
+  }
+
+  get expires_in() {
+    return Math.max.apply(null, [this.expiresAt - Date.now(), 0]);
```

Comment:
why are you using an array here?
why not `Math.max(this.expiresAt - Date.now(), 0)` ?

Reply:
copied from the lib^^